### DISCUSSION TITLE
Wiki page storage cannot be expanded

### DISCUSSION THREAD

**Discussion by Kurayami-Senpai** [requarks/wiki#6877] [Help / Questions]
I want to set the bodyParserLimit property in my config.yml instead.

like here https://github.com/requarks/wiki/blob/main/config.sample.yml#L147

But the problem is in my Wiki.js root directory /etc/wiki there is nothing except the .db secret.
I can't find any config.yml or master.yml
Do I have to create this myself if I have installed Wiki.js via Docker or should it actually be found there ?
Cause I can't find any config file

**Comment by fnafox** [comment]
If you've installed it using docker then the config file is in the container. If you want to expose the config then you need to mount it during container deployment.
https://docs.requarks.io/install/docker#alternative-mount-the-config-file

**Reply by Kurayami-Senpai** [reply]
Thanks for the information, I already thought that. I'm still fairly new to Docker so I wanted to ask before I actively start learning Docker by heart.
